Sources/Testing/Testing.docc/AssociatingBugs.md

```diff
@@ -82,3 +82,10 @@ The testing library defines several kinds of common bug/test relationship:
 
 <!-- Keep `.unspecified` as the last row above in order to imply it is a
 fallback. -->
+
+## Topics
+
+- <doc:BugIdentifiers>
```

Comment:
While you're considerations organizational improvements to these articles…  I wonder if things would be clearer if we consolidated `BugIdentifiers.md` into this article? I'd find it useful to have the details about `Bug` collected into one page, and it feels a little odd for this topic's content to be spread over two articles

Reply:
I'll leave that for a separate PR if that's okay? There was a reason we split this up originally, lost to time…